docs/content/users/usage/cli.md

```diff
@@ -69,17 +69,28 @@ To import static file assets for a project, such as uploaded images and document
 Other project types need a custom configuration to be able to use this command.
 
 ```bash
-ddev import-files
-Provide the path to the directory or archive you wish to import. Please note, if the destination directory exists, it will be replaced with the import assets specified here.
-Import path:
-~/Downloads/files.tar.gz
-Successfully imported files for drupal8
+$ ddev import-files
+Provide the path to the source directory or archive you wish to import.
+Please note: if the destination directory exists, it will be replaced with the
+import assets specified here.
```

Comment:
That's actually what it already says isn't it? 

> if the destination directory exists, it will be replaced with the import assets specified here

Reply:
aside the detail about content and contents i've asked over in discord.  

> If using a Tar or Zip archive, the archive should contain only the files inside the files directory. For example in a Drupal site with files at sites/default/files, the archive should **not* contain the files directory, just the contents of the files directory

the second sentence took me one or two times until i understood. i wonder if it would make sense to switch the order  to what the goal is and then what situation should be avoided. i suppose the wording could still be improved

```For example in a Drupal site with files at sites/default/files, the archive should only contain the contents of that `files directory`. You shouldn't end up with a single `files` directory on unarchive containing all the contents.```

And i wonder if it would make sense to shorten: 

> The easiest way to do this with the tar command is to cd into the directory and create the archive there

I would simply strike "the easiest way to do this.." and use something like 

```
To export the files directory use: 
```

or not sure what matt recommends as the common phrase for pointing at one or two lines of terminal input.